### DISCUSSION TITLE
Built-in support for Composer cache

### DISCUSSION THREAD

**Discussion by martijngastkemper** [shivammathur/setup-php#515] [Ideas / RFC] [answered]
Enabling caching for Composer requires quite some YAML lines. Inspired by the built-in cache support of [actions/setup-node](https://github.com/actions/setup-node#caching-packages-dependencies) I would like to have this feature for PHP as well.

Before looking into it and creating a PR I was wondering if this feature is already proposed, planned, or killed. I can't find any discussion about it, but maybe other channels are used. Can a maintainer elaborate on that?

**Comment by shivammathur** [comment, marked as answer]
This is planned for v3.

**Reply by martijngastkemper** [reply]
That's awesome. Thanks!
